iphone/TitaniumKit/TitaniumKit/Sources/API/TiAnimation.m

```diff
@@ -232,6 +240,28 @@ - (TiPoint *)center
   return center;
 }
 
+- (void)setColor:(id)color_
+{
+  [color release];
```

Comment:
```suggestion
  RELEASE_TO_NIL(color);
```

Does it matter if we call release on a nil pointer? We tend use the above everywhere to guard against it.